SINGLE-VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+# Validation - Single Safe
```

Comment:
can you update your directory name to be 007-opcm-upgrade and all references as per: 
![CleanShot 2025-04-10 at 10 06 53@2x](https://github.com/user-attachments/assets/74d686ec-19d9-4056-bd7a-ea4c4234935a)

Reply:
I also don't think you need the image added in this PR as the isRc slot isn't changed.

Reply:
e335107e962e

Reply:
The image is still in this PR, should it be?

Reply:
no, deleted it.